src/pages/resultsView/survival/SurvivalChart.tsx

```diff
@@ -45,6 +81,60 @@ export default class SurvivalChart extends React.Component<ISurvivalChartProps,
     private unalteredLegendText = 'Cases without Alteration(s) in Query Gene(s)';
     private svgContainer: any;
     private svgsaver = new SvgSaver();
+    private styleOptsDefaultProps:StyleOpts = {
+        width: 900,
+        height: 500,
+        padding: {top: 20, bottom: 50, left: 60, right: 20},
+        axis: {
+            x: {
+                ticks: {
+                    size: 8,
+                    stroke: 'black',
+                    strokeWidth: 1
+                },
+                tickLabels: {
+                    padding: 2, fill: "black"
+                },
+                axisLabel: {
+                    padding: 35, fill: "black"
+                },
+                grid: {opacity: 0},
+                axis: {stroke: "black", strokeWidth: 1}
+            },
+            y: {
+                ticks: {
+                    size: 8,
+                    stroke: 'black',
+                    strokeWidth: 1
+                },
+                tickLabels: {
+                    padding: 2, fill: "black"
+                },
+                axisLabel: {
+                    padding: 45, fill: "black"
+                },
+                grid: {opacity: 0},
+                axis: {stroke: "black", strokeWidth: 1}
+            }
+        },
+        legend: {
+            x: 600,
+            y: 40
+        }
+    };
+
+    @computed
+    get styleOpts() {
+        let configurableOpts: StyleOpts = _.merge(this.styleOptsDefaultProps, this.props.styleOpts);
```

Comment:
It turns out that _.merge method treats the first param as the "target."  You wouldn't think it would work this way but, the defaults object ends up getting overwritten.  So, it's a good practice to use _.merge({}, obj1, obj2).  That way obj1 doesn't get mutated.  Weird, I know.

Reply:
this probably doesn't matter in this case because you probably won't use defaults again in life of component, but in another circumstance you might.

Reply:
Yeah, the default should be immutable. I will change to the suggestion above.